### DISCUSSION TITLE
Chamilo Course Builder certification prep

### DISCUSSION THREAD

**Discussion by Moigaston** [chamilo/chamilo-lms#5885] [General] [answered]
Ce site est-il fiable ? https://www.udemy.com/course/chamilo-chabobu2014en/?couponCode=MTST7102224A2

Mais pourquoi le paiement est-il en livres sterling ?

Et l'examen est à payer en dollars! :)

on n' fait pas partie de l'Union Européenne? :)

P.S.: Et où sont passés les smileys ? Dans quelle monnaie faut-il payer pour en avoir ?
![image](https://github.com/user-attachments/assets/2844a66f-c735-47e7-b8da-6e2241bc92a0)

**Comment by ywarnier** [comment, marked as answer]
Bonjour @Moigaston 

Bah, c'est un vieux cours que j'avais mis en ligne du temps où Chamilo n'avait pas de plugin de ventes en ligne. Le cours est bien, mais il est assez vieux maintenant, et si je me souviens bien je n'avais pas encore de bon micro, donc le son est à mettre à fond pour entendre quelque chose (ce qui n'empêche rien).

Pour l'examen, tu peux le passer directement sur https://campus.chamilo.org (pas besoin de passer par Udemy). Le plugin de ventes de Chamilo ne permet de mettre qu'une monnaie courante, et à l'époque Chamilo était très populaire en Amérique Latine, où la monnaie d'échange (ils n'ont pas de monnaie unique comme nous) était et reste le dollar américain.

Tout ça devrait se remettre en place dans le courant de 2025 avec des cours et certifications mis à jour et faits directement sur Chamilo 2.0.

Si tu as des demandes/besoins particuliers, n'hésites pas à le faire savoir ici.

Tu peux éviter de payer le cours pour Chamilo 1.11 en étudiant directement le manuel en français ici : https://docs.chamilo.org/1.11.x-fr mais l'idée était de fournir un raccourci en vidéo pour passer la certification. Ça n'a jamais atteint le succès escompté, mais à l'époque notre communauté était plus petite. Peut être que ça vaudra l'effort en 2025... :wink:

**Comment by Moigaston** [comment]
Merci pour ta réponse , Yannick.
`Pour l'examen, tu peux le passer directement sur https://campus.chamilo.org/` J'y suis allé à l'instant, mais je ne vois que l'instance de Chamilo mise à disposition pour ceux qui veulent tester. Où est-ce que je trouve le lien pour passer l'examen?
J'utilise Chamilo depuis 2020 (prof., admin et formateur) et j'aimerais tenter l'examen.
